app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, in_file, out_file = command.split()
```

Comment:
To avoid an error related to unpacking a list that contains more than 3 elements after split, you could add the following syntax
```suggestion
    command, in_file, out_file, *_ = command.split()
```